view/next-project/src/components/sponsoractivities/EditModal.tsx

```diff
@@ -44,13 +60,17 @@ export default function EditModal(props: ModalProps) {
     await put(updateSponsorStyleUrl, data);
   };
 
+  const remarkCoupon = `【クーポン】[詳細 :  ○○],
+  【広告掲載内容】[企業名 : x],[住所 : x],[HP : x],[ロゴ : x],[営業時間 : x],[電話番号 : x],[キャッチコピー : x],[地図 : x],[その他 :  ]`;
+  const remarkElse = `【広告掲載内容】[企業名 : x],[住所 : x],[HP : x],[ロゴ : x],[営業時間 : x],[電話番号 : x],[キャッチコピー : x],[地図 : x],[その他 :  ]`;
```

Comment:
定数関係はグローバルな領域（24行目あたり）で定義したいな！
グローバルな領域だと、レンダリングのたびに読み込まれることはないので
あと定数は全部大文字＋スネークケースで！

```suggestion
  const REMARK_COUPON = `【クーポン】[詳細 :  ○○],
  【広告掲載内容】[企業名 : x],[住所 : x],[HP : x],[ロゴ : x],[営業時間 : x],[電話番号 : x],[キャッチコピー : x],[地図 : x],[その他 :  ]`;
  const REMARK_ELSE = `【広告掲載内容】[企業名 : x],[住所 : x],[HP : x],[ロゴ : x],[営業時間 : x],[電話番号 : x],[キャッチコピー : x],[地図 : x],[その他 :  ]`;
```

Reply:
了解!